packages/core/src/core/auth.ts

```diff
@@ -1,7 +1,8 @@
 import * as hapi from "@hapi/hapi";
 import open from "open";
 import { CONSTS, bold, underline, spinner, writeError } from "../utils";
-import { setAuthConfig } from "../filesystem";
+import { PrismicSharedConfigManager } from "../filesystem/PrismicSharedConfig";
```

Comment:
Why changing the name of this services ? As this file handle only the auth information, I found that sharedConfig doesn't help to understand what is inside the file.

Reply:
because actually it does more than this,  it handles the base which is not specifically auth oriented and same for the userId